scripts/blueprint.gd

```diff
@@ -90,6 +91,112 @@ enum {
 
 
 #region Public Variables
+#region Enum-likes
+static var all_types: Array[StringName] = [
+	&"None",
+	&"Minion",
+	&"Spell",
+	&"Weapon",
+	&"Hero",
+	&"Location",
+	&"Hero_power",
+]
+
+static var all_tribes: Array[StringName] = [
+	&"None",
+	&"Beast",
+	&"Demon",
+	&"Dragon",
+	&"Elemental",
+	&"Mech",
+	&"Murloc",
+	&"Naga",
+	&"Pirate",
+	&"Quilboar",
+	&"Totem",
+	&"Undead",
+	&"All",
+]
+
+static var all_spell_schools: Array[StringName] = [
+	&"None",
+	&"Arcane",
+	&"Fel",
+	&"Fire",
+	&"Frost",
+	&"Holy",
+	&"Nature",
+	&"Shadow",
+]
+
+static var all_rarities: Array[StringName] = [
+	&"Free",
+	&"Common",
+	&"Rare",
+	&"Epic",
+	&"Legendary",
+]
+
+static var all_rarity_colors: Dictionary = {
+	&"Free": Color.WHITE,
+	&"Common": Color.GRAY,
+	&"Rare": Color.BLUE,
+	&"Epic": Color.PURPLE,
+	&"Legendary": Color.GOLD,
+}
```

Comment:
The rarities will probably get their own module/modules. I just have them here for now, but in the future we would want to split them.

This applies to most stuff here. Splitting them out of core would be difficult though...